File: app/assets/javascripts/models/post.js
Blog.Post = DS.Model.extend(Blog.DatableMixin, {
    format: 'LL',
    published: DS.attr('boolean'),
    slug: DS.attr('string'),
    title: DS.attr('string'),
    description: DS.attr('string'),
    body: DS.attr('string'),
    publishedAt: DS.attr('string'),

    tags: DS.hasMany('tag'),
    comments: DS.hasMany('comment'),
    category: DS.belongsTo('category'),
    author: DS.belongsTo('author')
})

File: app/models/post.rb
class Post < ActiveRecord::Base
  extend FriendlyId
  friendly_id :title, use: :slugged

  # relationship
  has_and_belongs_to_many :tags
  has_many :comments
  belongs_to :author
  belongs_to :category

  # scope
  scope :recent, -> { order('published_at DESC') }
end